lib/helpers/unilogin.ts

```diff
@@ -13,9 +11,7 @@ export const getUniloginWsCredentials = () => {
 }
 
 export const getLibraryMunicipalityId = () => {
-  const municipalityId = process.env.UNILOGIN_MUNICIPALITY_ID ?? ""
-  if (!municipalityId) {
-    throw new Error("Missing Unilogin WS credentials")
-  }
+  const municipalityId = getServerEnv("UNILOGIN_MUNICIPALITY_ID")
+
   return municipalityId
 }
```

Comment:
Just a sugesstion. More a matter of taste. But since there is not a lot of meat/logic in this function I would write:
```typescript
export const getLibraryMunicipalityId = () => getServerEnv("UNILOGIN_MUNICIPALITY_ID")